src/widget.js

```diff
@@ -86,6 +86,11 @@ export default class Widget extends Plugin {
 		const viewDocument = editor.editing.view;
 		let element = domEventData.target;
 
+		// Widget is not selectable when editor is read-only.
```

Comment:
Widget might be salectable but only on the model side (like entire editor in read-only mode). But I think this selection should be not visible like caret is not visible when editor is read-only. WDYT?